docs/accessing-services.asciidoc

```diff
@@ -0,0 +1,157 @@
+[id="{p}-accessing-elastic-services"]
+= Accessing Elastic Stack services
+
+All Elastic Stack resources deployed by the ECK Operator are secured by default, which means it sets up basic authentication and TLS to encrypt network traffic to, from, and within your Elasticsearch cluster.
+
+Elasticsearch and Kibana are then accessible using native Kubernetes services that are not reachable from the public Internet by default.
+
+[id="{p}-kubernetes-service"]
+== Kubernetes service for each resource
+
+The operator manages a Kubernetes service for each resource (`Elasticsearch` or `Kibana`),  named `<name>-<kind>`, of `ClusterIP` type by default. `ClusterIP` exposes the service on a cluster-internal IP and makes the service only reachable from within the cluster.
+
+[source,sh]
+----
+> kubectl get svc
+
+NAME                TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)          AGE
+hulk-es             ClusterIP      10.19.252.160   <none>           9200:32361/TCP   25m
+hulk-kibana         ClusterIP      10.19.247.151   <none>           5601:32567/TCP   25m
+----
+
+[id="{p}-tls-certificates"]
+== TLS Certificates
+
+The operator manages a CA certificate for Elasticsearch as a `Secret` named `<name>-ca`. Supporting user defined CA certificates is on the roadmap link:https://github.com/elastic/cloud-on-k8s/issues/776[#776].
```

Comment:
Not sure if we should link from the website into GH issues (?)

Reply:
I think we could remove that from the doc, then add a section about how to use your own certs in that doc once available.

Reply:
Let's remove the link. I'm still very tempted to keep a note about supporting custom cert in the future to avoid questions.